src/di/container.ts

```diff
@@ -126,4 +132,20 @@ export const registerDependencies = (instance: FastifyInstance) => {
       pattern: CRON_3AM_MONDAY,
     })
     .build();
+
+  // Register EtherPad
+  const etherPadConfig = resolveDependency(EtherPadServiceConfig);
+  // connect to etherpad server
+  registerValue(
+    Etherpad,
+    wrapErrors(
+      new Etherpad({
+        url: etherPadConfig.url,
+        apiKey: etherPadConfig.apiKey,
+        apiVersion: etherPadConfig.apiVersion,
+      }),
+    ),
+  );
```

Comment:
I think we need to discuss about it, because I probably don't fully understand the use of this `wrapErrors` function. But I think we should handle it with custom `decorators` instead (Or any other suggestion). So we can get rid of this call.

Reply:
Here I just moved the existing code, so I didn't refactor the wrapErrors. I'll try to improve it if I can